charts/sonarqube/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 All changes to this chart will be documented in this file.
 
 ## [10.6.0]
+
```

Comment:
I inserted the blank-lines because of [MD-022](https://github.com/DavidAnson/markdownlint/blob/v0.34.0/doc/md022.md).

I just removed them now and we can discuss this later.